packages/cf-component-radio/src/RadioTheme.js

```diff
@@ -0,0 +1,15 @@
+export default baseTheme => ({
+  cursor: 'pointer',
+  display: 'block',
+  minHeight: '1em',
+  paddingLeft: '2em',
+  marginTop: 'initial',
+
+  '&:first-child': {
+    marginTop: 'initial'
+  },
+
+  '&:hover input': {
+    borderColor: '#256298'
```

Comment:
If this color is only used for borders then it's fine to use Deepsea. If it's used for hovers as well then the colors (`#256298` and `#1E4E79` are probably too far apart (and should be added to the palette).